src/components/index.ts

```diff
@@ -6,6 +6,7 @@ export { default as Grid } from './grid';
 export { default as Input } from './input';
 export { default as Link } from './link';
 export { default as Section } from './section';
+export { default as SimpleTitleHeading } from './simpleTitleHeading';
```

Comment:
Maybe just 
```suggestion
export { default as SimpleHeader } from './simpleHeader';
```
?
That's how we referred to this component before... What do you think?

Reply:
I think, it is better your suggestion, more compact. I will change it and sync the name with the designers as well. Thanks!